### ISSUE TITLE
Add Entity and EntitySet attributes to the sphinx documents

### ISSUE THREAD

**Issue by rwedge** [FeatureLabs/featuretools#339]
### Bug/Feature Request Description
The [EntitySet](https://docs.featuretools.com/generated/featuretools.EntitySet.html#featuretools.EntitySet) and [Entity](https://docs.featuretools.com/generated/featuretools.Entity.html#featuretools.Entity) documentation pages only have partial documentation of their respective class attributes, found at the bottom of each page.  Adding attribute documentation for these two classes in a way that sphinx can parse nicely would be useful.

-----
*Issues created here on Github are for bugs or feature requests. For usage questions and questions about errors, please ask on Stack Overflow with the [featuretools](https://stackoverflow.com/questions/tagged/featuretools) tag. Check the [documentation](https://docs.featuretools.com/#get-help) for further guidance on where to ask your question.*

**Comment by jeff-hernandez** [comment]
When using the `autosummary` extension in Sphinx, it seems like there is not a clear distinction between class attributes and instance attributes. Attributes defined using the `@property` decorator seem to be the only attributes included in the summary table at the bottom of the page. One option could be to manually define these attributes to render in the class description.
```
"""
Stores all actual data for a entityset

Attributes:
    id (str) : Unique identifier to associate with this instance
    entity_dict
    relationships (list[(str, str, str, str)]): List of relationships between entities.
    time_type

Properties:
    metadata

"""
```

**Comment by rwedge** [comment]
Addressed by #349
